sonic_data_client/db_client.go

```diff
@@ -91,19 +91,19 @@ func (val Value) Compare(other queue.Item) int {
 }
 
 type DbClient struct {
-	prefix	*gnmipb.Path
+	prefix  *gnmipb.Path
 	pathG2S map[*gnmipb.Path][]tablePath
 
-	q		*queue.PriorityQueue
```

Comment:
is all formatting change coming from gofmt? Have you applied it?

Reply:
Applying now as apart of final changes. Push coming soon.